<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VertebraRegistrationWidget</class>
 <widget class="QWidget" name="VertebraRegistrationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>898</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>642</width>
     <height>903</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="refreshButton">
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="addFrameButton">
      <property name="text">
       <string>Add frame</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="sourceImageLabel">
          <property name="maximumSize">
           <size>
            <width>170</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Moving Volume:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="sourceImageComboBox">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="transformObjectLabel">
          <property name="maximumSize">
           <size>
            <width>170</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>CT Volume:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="targetImageComboBox">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QCheckBox" name="debugCheckBox">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Debug</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Number of frames per volume</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Beta</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Gamma</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QSpinBox" name="numberOfFramesSpinBox">
              <property name="minimum">
               <number>5</number>
              </property>
              <property name="maximum">
               <number>200</number>
              </property>
              <property name="singleStep">
               <number>5</number>
              </property>
              <property name="value">
               <number>25</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="betaSpinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>0.400000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="gammaSpinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
              <property name="value">
               <double>2.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="liveCheckBox">
      <property name="text">
       <string>Acuiring live images</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QVTKWidget" name="usImageWindow" native="true">
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="debugTextBrowser"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
